lib/client.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+//jshint latedef: nofunc
```

Comment:
We have a circular dependency on the functions we are using.. It does not allow the function declaration after being called.